inference-engine/tests/functional/inference_engine/ngraph_reader/ctc_greedy_decoder_tests.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadCTCGreedyDecoderNetwork) {
+    std::string model = R"V0G0N(
+<net name="ctcgreedydecoder" version="10">
+	<layers>
+		<layer id="0" name="CTCGreedyDecoder/placeholder_port_0" type="Parameter" version="opset1">
+			<data shape="20,8,128" element_type="f32"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>20</dim>
+					<dim>8</dim>
+					<dim>128</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="1" name="Fill/Broadcast/Output_0/Data__const" type="Const" version="opset1">
+			<data offset="0" size="640" shape="20,8" element_type="f32"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>20</dim>
+					<dim>8</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="2" name="CTCGreedyDecoder" type="CTCGreedyDecoder" version="opset1">
+			<data ctc_merge_repeated="1"/>
```

Comment:
I think we shall add another test case when attribute merge_repeated=false is defined.